fluttertest.nix

```diff
@@ -0,0 +1,27 @@
+with (import ./. {});
+
+callPackage (
+
+{ flutter
+, fetchFromGitHub
+}:
+
+flutter.mkFlutterApp {
+  pname = "firmware-updater";
+  version = "unstable";
+
+  # this is broken, you need to run it and put in your own hash here
+  vendorHash = "sha256-BS2I9ILEQ5EAzjg/qTE/znIheveI7DnYBUXh6n2mQc8=";
+
+  src = fetchFromGitHub {
+    owner = "canonical";
+    repo = "ubuntu-desktop-installer";
+    rev = "f7340e75518b357a2f7726cfadd62853dd6ff865";
+    sha256 = "la+D8ifDQKJFYDuHZas1mjSqaZgz+jqKag7xhDIk2CM=";
+    fetchSubmodules = false;
+  };
+
+  sourceRoot = "source/packages/ubuntu_desktop_installer";
+}
+
+) {}
```

Comment:
This file should also not be included!

Reply:
it's specifically for testing, just until I get else working, then I'll throw it out

Reply:
You can have a file locally but not committed in git

Reply:
I know, but it's in the pr desc for testing, as otherwise how would you test it? I'm currently packaging another app that makes sense to be in nixpkgs, but that's still WIP